examples/deployment/docker-compose.yml

```diff
@@ -3,37 +3,54 @@ services:
   mysql:
     image: mariadb:10.1
     environment:
-      - MYSQL_ROOT_PASSWORD=$DB_PASSWORD
+      - MYSQL_ROOT_PASSWORD
   trillian-db-seed:
```

Comment:
Should `trillian-db-seed` depend on `mysql` ?

Reply:
Done.